README.md

```diff
@@ -6,7 +6,7 @@
 
 <p align="center">
     <a href="https://www.python.org/downloads/release/python-370/" title="python version"><img src="https://img.shields.io/badge/Python-3.7%2B-blue.svg"></a>
-    <a href="https://travis-ci.com/cap-ntu/ML-Model-CI" title="Build Status"><img src="https://travis-ci.com/cap-ntu/ML-Model-CI.svg?token=SvqJmaGbqAbwcc7DNkD2&branch=master"></a>
+    <a href="https://github.com/cap-ntu/ML-Model-CI/actions" title="Build Status"><img src="https://github.com/cap-ntu/ML-Model-CI/actions/workflows/run_test.yml/badge.svg?branch=master"></a>
```

Comment:
seems the badge has not been set properly

Reply:
The badge will be valid if this pr is merged, as there is no corresponding yml file currently.
Here is a preview link with my forked repo:
https://github.com/univerone/ML-Model-CI/actions/workflows/test.yml/badge.svg?branch=add-github-action